partshop-backend/src/main/java/hu/kosztaauto/partshop/service/AlkatreszService.java

```diff
@@ -29,6 +31,18 @@ public List<AlkatreszWithWarehouseDTO> getAllWarehousesByItem() {
                 .collect(Collectors.toList());
     }
 
+    public AlkatreszDTO getItemById(String id) {
+        return alkatreszRepository.findById(id)
+                .map(this::convertToAlkatreszDTO)
+                .orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND, "Item not found"));
+    }
+
+    public List<AlkatreszDTO> searchByName(String name) {
```

Comment:
Ezt nem a frontenden kellene kezelnünk?

Reply:
Hát ott is kezelheted, nem akadályoz meg benne, de át is írhatom